docs/user-guide/walkthrough/index.md

```diff
@@ -11,36 +11,36 @@ For Kubernetes 101, we will cover kubectl, pods, volumes, and multiple container
 
 {% include task-tutorial-prereqs.md %}
 
-In order for the kubectl usage examples to work, make sure you have an example directory locally, either from [a release](https://github.com/kubernetes/kubernetes/releases) or [the source](https://github.com/kubernetes/kubernetes).
+In order for the kubectl usage examples to work, make sure you have an example directory locally, either from [a release](https://github.com/kubernetes/kubernetes/releases) or the latest .yaml files located at: https://github.com/kubernetes/website/tree/master/docs/user-guide/walkthrough.
 
 * TOC
 {:toc}
 
 
 ## Kubectl CLI
 
-The easiest way to interact with Kubernetes is via the [kubectl](/docs/user-guide/kubectl-overview/) command-line interface.
+The easiest way to interact with Kubernetes is through the [kubectl](/docs/reference/kubectl/overview/) command-line interface.
 
-For more info about kubectl, including its usage, commands, and parameters, see the [kubectl CLI reference](/docs/user-guide/kubectl-overview/).
+For more info about kubectl, including its usage, commands, and parameters, see [Overview of kubectl](/docs/reference/kubectl/overview/).
```

Comment:
Probably don't need this sentence. It just repeats the link from the previous sentence.